django_ipam/base/models.py

```diff
@@ -9,9 +9,11 @@
 
 from .fields import NetworkField
 
+HELP_TEXT = 'Enter valid CIDR network field, for example - IPv4: 10.0.0.0/24 or IPv6: fdb6:21b:a477::9f7/64'
+
 
 class AbstractSubnet(TimeStampedEditableModel):
-    subnet = NetworkField(default="10.0.0.0/24", db_index=True)
+    subnet = NetworkField(help_text=HELP_TEXT, db_index=True)
```

Comment:
Good improvement, let's improve it even more, do this and get rid of the `HELP_TEXT` variable above (for consistency with the other ow2 modules):

```python
class AbstractSubnet(TimeStampedEditableModel):
    subnet = NetworkField(db_index=True,
                          help_text=_('Subnet in CIDR notation, eg: "10.0.0.0/24" '
                                      'for IPv4 and "fdb6:21b:a477::9f7/64" for IPv6'))
```